apps/web/app/networking/posts.api.ts

```diff
@@ -0,0 +1,280 @@
+import { createWebApiClient, defineApiConfig } from "./client";
+import type { PostDto } from "@/toolkit/dto/post.dto";
+
+export const CreatePostApi = defineApiConfig((request) => {
```

Comment:
Since this is a standard function, you should use camelCase instead of PascalCase